python/ucxx/ucxx/core.py

```diff
@@ -35,6 +35,7 @@ def init(
     progress_mode=None,
     enable_delayed_submission=None,
     enable_python_future=None,
+    connect_timeout=None,
```

Comment:
Can you add the missing docstring too? See https://github.com/rapidsai/ucx-py/blob/75c49e661947d55efda3f2bcdf6d72bb1bf01e91/ucp/core.py#L933-L936 for reference.